### PR TITLE
[Spanish] localize 'Digital Certificate

### PR DESCRIPTION
**Pull request by vasucp1207**
Signed-off-by: Vasu Singh <vasucp1207@gmail.com>

### Describe your changes
[Spanish] localize 'Digital Certificate

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/digital-certificate.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+título: Certificado Digital
+estado: Comentarios apreciados
+categoría: tecnología
```
Comment (electrocucaracha) [inline content/es/digital-certificate.md:4]:
```suggestion
title: Certificado Digital
status: Feedback Appreciated
category: Tecnología
```

File: content/es/digital-certificate.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+---
+
+## Lo que es
```
Comment (electrocucaracha) [inline content/es/digital-certificate.md:7]:
```suggestion
## ¿Qué es?
```

File: content/es/digital-certificate.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Los certificados nos permiten saber que la entidad particular con la que nos estamos comunicando es quien dice ser.
+También nos permiten asegurarnos de que nuestras comunicaciones sean privadas al cifrar los datos que enviamos y recibimos.
+
+## Problema que aborda
+
+Cuando los dispositivos se comunican a través de una red, no existe una garantía inherente de que un dispositivo en particular sea quien dice ser.
+Además, no podemos garantizar que el tráfico entre dos dispositivos no sea interceptado por un tercero.
+En consecuencia, cualquier comunicación puede ser potencialmente interceptada, comprometiendo información confidencial como nombres de usuario y contraseñas.
+
+## Cómo ayuda
```
Comment (electrocucaracha) [inline content/es/digital-certificate.md:19]:
```suggestion
## ¿Cómo ayuda?
```

File: content/es/digital-certificate.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+título: Certificado Digital
+estado: Comentarios apreciados
+categoría: tecnología
+---
+
+## Lo que es
+
+Un certificado (digital), también conocido como certificado de clave pública o certificado SSL, es un documento digital que se utiliza para ayudar a proteger las comunicaciones a través de la red.
```
Comment (electrocucaracha) [inline content/es/digital-certificate.md:9]:
```suggestion
Un certificado (digital), también conocido como certificado de clave pública o certificado SSL, es un documento digital utilizado para ayudar a proteger las comunicaciones que ocurren a través de la red.
```

File: content/es/digital-certificate.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+Los certificados nos permiten saber que la entidad particular con la que nos estamos comunicando es quien dice ser.
```
Comment (electrocucaracha) [inline content/es/digital-certificate.md:10]:
```suggestion
Los certificados nos permiten conocer la entidad en particular con la que nos estamos comunicando y si es quien dice ser.
```

File: content/es/digital-certificate.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+## Problema que aborda
+
+Cuando los dispositivos se comunican a través de una red, no existe una garantía inherente de que un dispositivo en particular sea quien dice ser.
+Además, no podemos garantizar que el tráfico entre dos dispositivos no sea interceptado por un tercero.
+En consecuencia, cualquier comunicación puede ser potencialmente interceptada, comprometiendo información confidencial como nombres de usuario y contraseñas.
+
+## Cómo ayuda
+
+Los clientes de correo electrónico modernos que utilizan certificados pueden notificarle si la identidad del remitente es correcta, al igual que los navegadores web (observe el pequeño candado frente a la barra de direcciones de su navegador web).
```
Comment (electrocucaracha) [inline content/es/digital-certificate.md:21]:
```suggestion
Los clientes de correo electrónico modernos que utilizan certificados pueden notificar si la identidad del remitente es correcta, al igual que los navegadores web (vea el pequeño candado frente a la barra de direcciones en su navegador web).
```

File: content/es/digital-certificate.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+Por otro lado, los certificados se pueden usar para cifrar la comunicación entre entidades en Internet.
+Proporcionan una técnica de encriptación que hace que sea casi imposible que alguien que intercepte la comunicación pueda leer los datos.
```
Comment (electrocucaracha) [inline content/es/digital-certificate.md:23]:
```suggestion
Proporcionan una técnica de encriptación que hace que sea casi imposible que alguien que intercepte la comunicación y pueda leer los datos.

## Términos relacionados
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Review by krol3 — APPROVED** [review]
lgtm!  @vasucp1207  Please fix the Spellcheck, you need to add the word `ssl`

**Comment by vasucp1207** [comment]
> lgtm! @vasucp1207 Please fix the Spellcheck, you need to add the word `ssl`

I do not get it can you please specify more about it?

**Comment by electrocucaracha** [comment]
> > lgtm! @vasucp1207 Please fix the Spellcheck, you need to add the word `ssl`
> 
> I do not get it can you please specify more about it?

There is a CI task that checks for spelling errors in the Spanish content. The task extends its internal dictionary with more words defined in this [list](https://github.com/cncf/glossary/blob/dev-es/content/es/.wordlist.txt). This PR requires the addition of new words in that list, otherwise CI [task won't pass](https://github.com/cncf/glossary/actions/runs/3607532915/jobs/6079607326#step:5:13)

Could you include those words into that file? And also make sure that they are added in [alphabetical order](https://github.com/cncf/glossary/blob/dev-es/.github/workflows/es-spellcheck.yml#L27-L30).

**Comment by vasucp1207** [comment]
@electrocucaracha, I updated the wordlist file when that merges then I also update this PR.

**Comment by electrocucaracha** [comment]
> @electrocucaracha, I updated the wordlist file when that merges then I also update this PR.

You have to include those changes in this PR to make the Spellchecker CI pass.

**Comment by vasucp1207** [comment]
@electrocucaracha, check now.

**Comment by vasucp1207** [comment]
@electrocucaracha, check now.

File: content/es/digital-certificate.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+Cuando los dispositivos se comunican a través de una red, no existe una garantía inherente de que un dispositivo en particular sea quien dice ser.
+Además, no podemos garantizar que el tráfico entre dos dispositivos no sea interceptado por un tercero.
+En consecuencia, cualquier comunicación puede ser potencialmente interceptada, comprometiendo información confidencial como nombres de usuario y contraseñas.
+
+## ¿Cómo ayuda?
+
+Los clientes de correo electrónico modernos que utilizan certificados pueden notificar si la identidad del remitente es correcta, al igual que los navegadores web (vea el pequeño candado frente a la barra de direcciones en su navegador web).
+Por otro lado, los certificados se pueden usar para cifrar la comunicación entre entidades en Internet.
+Proporcionan una técnica de cifrada que hace que sea casi imposible que alguien que intercepte la comunicación y pueda leer los datos.
```
Comment (92nqb) [inline content/es/digital-certificate.md:23]:
```suggestion
Proporcionan una técnica de cifrado que hace que sea casi imposible que alguien que intercepte la comunicación y pueda leer los datos.
```

**Review by 92nqb — CHANGES_REQUESTED** [review]

**Review by 92nqb — APPROVED** [review]
lgtm

**Pull request merged**